README.md

```diff
@@ -1 +1,11 @@
+# Project Filler CLI
 
+## What is it ?
+
+This project aims to help developers to fill most programming projects (JavaScript, TypeScript, Angular, JAVA..) with tools and good practices proposed by Zenika. It is a command line interface that will detect your project's stack and will search for rules to apply corresponding to this stack.
+
+## How can I use it ?
+
+For the moment, the project is not really usable, but you can try it using the binary release corresponding to your platform. Download the release tarball, decompress it, and run the bin/project-starter-cli binary in your shell like this :
```

Comment:
What is the URL of the binary ?
You can add a Getting Started in order to explain how to launch the project from scratch as a developer.